msteams-platform/graph-api/rsc/resource-specific-consent.md

```diff
@@ -141,9 +204,27 @@ Once the app has been installed to a team, you can use [Graph Explorer](https://
 > - Select **Get link to team**.
 > - Copy and save the **groupId** value from the string.
 > - Log into **Graph Explorer**.
-> - Make a **GET** call to the following endpoint: `https://graph.microsoft.com/beta/groups/{teamGroupId}/permissionGrants`. The clientAppId field in the response will map to the appId specified in the Teams app manifest.
+> - Make a **GET** call to the following endpoint: `https://graph.microsoft.com/beta/groups/{teamGroupId}/permissionGrants`. The `clientAppId` field in the response will map to the `webApplicationInfo.id` specified in the Teams app manifest.
   ![Graph explorer response to GET call.](../../assets/images/graph-permissions.png)
 
+#### See also
+[Get the names and other details of apps installed in the specified team](/graph/api/team-list-installedapps)
+
+### Check your app for added RSC permissions in a chat
+
+> [!div class="checklist"]
+>
+>- Get the chat thread ID from the Teams *web* client.
+> - In the Teams web client, select **Chat** from the far left nav bar.
+> - Select the chat where the app is installed from the drop-down menu.
+> - Copy the web URL and save the chat thread ID from the string.
+![Chat thread id from web URL.](../../assets/images/chat-thread-id.png)
+> - Log into **Graph Explorer**.
+> - Make a **GET** call to the following endpoint: `https://graph.microsoft.com/beta/chats/{chatId}/permissionGrants`. The `clientAppId` field in the response will map to the `webApplicationInfo.id` specified in the Teams app manifest.
+
+#### See also
+[Get the names and other details of apps installed in the specified chat](/graph/api/chat-list-installedapps#example-2-get-the-names-and-other-details-of-apps-installed-in-the-specified-chat)
```

Comment:
"For information about how to get details about apps installed in a specific chat, see <topic title>."